tap_pendo/streams.py

```diff
@@ -206,14 +75,31 @@ def update_currently_syncing(state, stream_name):
 class Server42xRateLimitError(Exception):
     pass
 
+
+class Endpoints:
```

Comment:
@namrata270998 Add required unit test cases for these code changes

Reply:
Done